### ISSUE TITLE
Multiple Package Support

### ISSUE THREAD

**Issue by jkerry** [test-kitchen/kitchen-habitat#22]
In order to accurately simulate production systems we need to be able to install multiple packages in individual supervisors.

**Comment by jkerry** [comment]
I'll need to dig in a bit, but I imagine we can just encapsulate that behavior (install_latest_artifact) and iterate over it

**Comment by skylerto** [comment]
Make sense, so it's either all or none in that case?

**Comment by jkerry** [comment]
Yeah, probably to start, then maybe look into service specific overrides

**Comment by skylerto** [comment]
seems like the most feasible.
